johnsonandjohnson/Bodiless-JS
# fix(vital-navigation): fix breadcrumb axe.

## Changes
Switch Separators to CSS only so accessible.
Fixed bug that Page was using Generic instead of Default.  FYI Default == Generic.

## Test Instructions
* Verify breadcrumb still renders with some visually accepable divider. 
* Verify Default page template renders identically.

* https://bodiless-js-next-git-fix-breadcrumb-jnj-demo-account.vercel.app/styleguide/generic-template/
